requirements.txt

```diff
@@ -1,3 +1,7 @@
+matplotlib==3.8.0
+numpy==1.24.4
 pandas==2.2.3
-matplotlib==3.9.2
-numpy==2.1.2
```

Comment:
This version of numpy is not available for python 3.12, so we need to add to the README that they need to make the pyenv based off of python 3.11 if they want to download all dependencies

Reply:
Hm i just saw that the numpy im using is even older